nasa/cumulus-dashboard
# [CUMULUS-1758] Use react-table for resizing columns

This refactors the table implementation and allows for resizable columns.

It pulls in react-table for the underlying functionality and changes all table elements to be divs with classes matching the corresponding element. 

To test: 
Go to each table and make sure layout, selection, sorting, etc works as expected.
